CMakeLists.txt

```diff
@@ -387,29 +387,26 @@ endforeach()
 
 # Find Lua interpreter
 include(LuaHelpers)
-set(LUA_DEPENDENCIES lpeg mpack bit)
 if(NOT LUA_PRG)
   foreach(CURRENT_LUA_PRG luajit lua5.1 lua5.2 lua)
     # If LUA_PRG is set find_program() will not search
     unset(LUA_PRG CACHE)
-    unset(LUA_PRG_WORKS)
     find_program(LUA_PRG ${CURRENT_LUA_PRG})
 
     if(LUA_PRG)
-      check_lua_deps(${LUA_PRG} "${LUA_DEPENDENCIES}" LUA_PRG_WORKS)
```

Comment:
Something just occurred to me when I came back to this...being relatively unfamiliar with lua, I'm curious how often it is that lua packages end up being version-dependent? I ask because at least for the platform I am using at the moment, debian 9, the available lua packages seem to be independent of lua version.